modules/service/src/main/scala/ch/epfl/bluebrain/nexus/kg/service/routes/InstanceRoutes.scala

```diff
@@ -57,191 +54,151 @@ class InstanceRoutes(
 
   def routes: Route = handleExceptions(exceptionHandler) {
     handleRejections(RejectionHandling.rejectionHandler) {
-      pathPrefix("data" / Segment) { orgIdString =>
-        val orgId = OrgId(orgIdString)
-        pathEndOrSingleSlash {
-          (get & paginatedAndFiltered) { (pagination, filterOpt) =>
-            traceName("listInstancesOfOrg") {
-              parameter('deprecated.as[Boolean].?) { deprecated =>
-                val filter = Filter(deprecatedAndRev(deprecated)) and filterOpt.map(_.expr)
-                buildResponse(instanceQueries.list(orgId, filter, pagination), pagination)
-              }
-            }
+      pathPrefix("data") {
+        (pathEndOrSingleSlash & get & searchQueryParams) { (pagination, filterOpt, termOpt, deprecatedOpt) =>
+          traceName("listInstances") {
+            val filter = filterFrom(deprecatedOpt, filterOpt, querySettings.nexusVocBase)
+            instanceQueries.list(filter, pagination, termOpt) buildResponse(base, pagination)
```

Comment:
I would recommend avoiding the use of infixes for these function shapes.

Reply:
Done